tests/gtest/test_population_Recruitment.cpp

```diff
@@ -8,70 +8,49 @@ namespace
 
     TEST_F(PopulationPrepareTestFixture, CalculateRecruitment_works)
     {
-        // calculating recruitment for year 4
-        int year = 4;
-        int age = 0; // just calculating for age 0 
+        // calculating spawning biomass for year 4
+        int sb_year = 4;
+        int sb_age = 0;
+        int sb_i_age_year = sb_year * population.nages + sb_age;
+        int sb_i_agem1_yearm1 = (sb_year - 1) * population.nages + sb_age - 1;
+
+        population.CalculateMortality(sb_i_age_year, sb_year, sb_age);
+        population.CalculateNumbersAA(sb_i_age_year, sb_i_agem1_yearm1, sb_age);
+        for (size_t year = 0; year < nyears; year++) {
+           for (size_t age = 0; age < nages; age++){
+               int i_age_year = year * population.nages + age;
+               population.CalculateMaturityAA(i_age_year, age);
+           }
+        }
+        population.CalculateSpawningBiomass(sb_i_age_year, sb_year, sb_age);
+
+        // calculating phi0
+        double phi0 = population.CalculateSBPR0();
+    
+        // calculating recruitment for year 5
+        int r_year = 5;
+        int r_age = 0; // just calculating for age 0 
+        int r_i_age_year = r_year * population.nages + r_age;
+        int r_i_agem1_yearm1 = (r_year - 1) * population.nages + r_age - 1;
```

Comment:
Is this line needed? I don't see `r_i_agem1_yearm1` being used anywhere else in the file.

Reply:
No, `r_i_agem1_yearm1` is not needed. I have removed this line of code. Thanks for catching it.